packages/atlas/src/views/viewer/CategoryView/CategoryView.tsx

```diff
@@ -28,25 +31,22 @@ export const CategoryView = () => {
   const mdBreakpointMatch = useMediaMatch('md')
   const { id = '' } = useParams()
 
-  const { categories, totalVideosCount, loading } = useCategories()
+  const { totalVideosCount, loading } = useQnCategories()
+  const videoCategories = useVideoCategoriesWithCounter()
 
-  const mappedVideoCategories = categories?.map((category) => ({
-    ...videoCategories[category.id],
-    ...category,
-  }))
-  const otherCategory: Array<VideoCategoryData & VideoCategoryFieldsFragment> = useMemo(
+  const otherCategory: Array<DisplayCategory & VideoCategoryFieldsFragment> = useMemo(
     () =>
       sampleSize(
-        mappedVideoCategories?.filter((category) => category.id !== id),
+        videoCategories?.filter((category) => category.id !== id),
         3
       ),
-    [id, mappedVideoCategories]
+    [id, videoCategories]
   )
-  const currentCategory = mappedVideoCategories?.find((category) => category.id === id)
+  const currentCategory = findDisplayCategory(id)
 
   const headTags = useHeadTags(currentCategory?.name)
 
-  const { categoriesFeaturedVideos } = useCategoriesFeaturedVideos(id)
+  const { categoriesFeaturedVideos } = useCategoriesFeaturedVideos(sample(currentCategory?.videoCategories) || '')
```

Comment:
`useCategoriesFeaturedVideos` should just use display category IDs going forward

Reply:
At first, I wasn't sure what do you mean but are you sure about this? `useCategoriesFeaturedVideos` is an api hook